<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-S5GH85J" name="testcases.Sheet3_SmartGift_CompleteFlow" tests="2" failures="0" timestamp="11 Feb 2019 17:28:10 GMT" time="209.572" errors="1">
  <testcase name="giftflow" time="209.516" classname="testcases.Sheet3_SmartGift_CompleteFlow"/>
  <testcase name="zipFolder" time="0.056" classname="testcases.Sheet3_SmartGift_CompleteFlow">
    <error type="org.zeroturnaround.zip.ZipException" message="Given file &#039;.\reports\imagesnew&#039; doesn&#039;t exist!">
      <![CDATA[org.zeroturnaround.zip.ZipException: Given file '.\reports\imagesnew' doesn't exist!
at org.zeroturnaround.zip.ZipUtil.pack(ZipUtil.java:1622)
at org.zeroturnaround.zip.ZipUtil.pack(ZipUtil.java:1419)
at org.zeroturnaround.zip.ZipUtil.pack(ZipUtil.java:1402)
at utils.Reporter.zipFolder(Reporter.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:483)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:425)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- zipFolder -->
</testsuite> <!-- testcases.Sheet3_SmartGift_CompleteFlow -->
